mm/2s2h/Rando/Menu.cpp

```diff
@@ -181,9 +181,14 @@ static void DrawLogicConditionsTab() {
     UIWidgets::CVarSliderInt("Majora Access Remains Required",
                              Rando::StaticData::Options[RO_ACCESS_MAJORA_REMAINS_COUNT].cvar,
                              IntSliderOptions().Min(0).Max(4).DefaultValue(0));
+    UIWidgets::CVarSliderInt("Majora Access Masks Required",
+                             Rando::StaticData::Options[RO_ACCESS_MAJORA_MASKS_COUNT].cvar,
+                             IntSliderOptions().Min(0).Max(20).DefaultValue(0));
     UIWidgets::CVarSliderInt("Moon Access Remains Required",
                              Rando::StaticData::Options[RO_ACCESS_MOON_REMAINS_COUNT].cvar,
                              IntSliderOptions().Min(0).Max(4).DefaultValue(4));
+    UIWidgets::CVarSliderInt("Moon Access Masks Required", Rando::StaticData::Options[RO_ACCESS_MOON_MASKS_COUNT].cvar,
+                             IntSliderOptions().Min(0).Max(20).DefaultValue(20));
```

Comment:
I feel like the default should be 0, as moon access is not normally gated by masks at all.
```suggestion
                             IntSliderOptions().Min(0).Max(20).DefaultValue(0));
```

Reply:
Cool makes sense